packages/react-components/src/components/styles/TypingIndicator.styles.ts

```diff
@@ -17,7 +17,10 @@ export const typingIndicatorStringStyle = mergeStyles({
   fontWeight: 400,
   whiteSpace: 'pre',
   display: 'flex',
-  alignItems: 'center'
+  alignItems: 'center',
+  maxWidth: '41.25rem',
```

Comment:
same here; typing indicator, messagethread and inputBox all have this property seperately. (If we keep the max width) move to a container and have these fill the containers width